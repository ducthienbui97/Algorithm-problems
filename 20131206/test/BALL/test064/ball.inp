8 308737 607364 539049 568843
184204 364788 872003 990450 875343 212849 434231 684077 143451 859096 758576 998760 661437 16852 15825 92927 526226 394727 614761 15554 938468 708819 45362 25579 208391 772274 423731 304447 793392 540554 451664 665408 861095 751135 71614 232040 776873 243877 436308 332576 888313 852884 216010 871883 643910 532266 603500 801444 373316 474864 303101 135909 692296 173206 467257 463755 421366 724937 501620 432000 481807 298263 907385 574070 
617878 165521 128378 430955 775223 114325 167025 571568 315167 567828 472191 835828 488355 614827 186316 370591 282136 4414 243485 894298 456418 323273 272184 148985 821566 998431 325335 639080 830914 330582 367797 926865 924541 257593 524888 192324 900416 346855 274890 258013 792521 959636 948545 636997 303565 192416 740676 780260 945473 710009 501496 992619 655723 400277 981270 886414 612628 369359 474130 109598 670966 411384 103687 497738 
52006 84404 732493 202558 159303 309089 540194 649709 828346 932688 47862 50611 700503 337758 786225 316692 9152 774075 137290 169747 349148 144515 675109 173704 188204 1284 312843 645152 930256 604208 826438 59682 733946 478383 853493 168321 782250 793853 743698 649677 996095 546567 927722 774178 561348 877693 423144 468880 173012 447867 640931 640862 158885 202674 337233 445106 596848 369397 258929 340776 676198 521178 252219 506165 
474580 287109 449254 483777 428283 163373 838242 20912 776105 114288 872499 698950 19859 412162 444617 351637 172657 454100 208178 229007 961943 751055 421523 153427 98287 530772 549199 56106 234914 563631 901271 272795 414513 540862 654232 689988 495815 771669 178532 74219 209652 699915 264259 889053 275459 779542 6245 52411 529365 951427 150532 186164 524064 482104 773923 551260 888792 626578 801849 794928 630576 821377 194254 761869 566578 464620 172116 425102 76903 656401 761376 266342 523035 28782 237517 699119 772482 999449 636731 210285 200699 842142 253790 582208 675822 346413 988988 995223 941677 559610 729724 803046 184437 434082 630663 910047 860907 859931 584775 711576 34876 627427 418302 884569 366830 662011 130398 475539 143630 766757 472041 738742 954699 172636 953751 48988 16558 938147 535987 235890 260445 787581 293102 285610 346986 983064 492121 730702 346129 988829 98605 764577 335231 10134 836654 926264 212331 651180 252498 629744 18298 957056 839759 746769 965554 189485 935312 234922 112424 959076 920310 822196 89646 763312 91232 13740 709889 590534 789667 541724 879961 972891 409278 202858 301369 58219 72350 673667 330499 726096 705379 450620 604906 469311 327471 159834 581279 948900 496910 598989 117019 968304 85743 727167 419293 884722 690545 939057 265057 643873 876407 574519 394781 444988 702718 323721 491872 566257 885694 98496 383365 866684 627327 152238 63348 754204 946930 112078 709084 763439 828409 517745 323292 520904 807605 964858 941209 433631 746078 44118 13360 123127 556828 492957 883353 549173 888770 359376 104365 526691 263913 902512 844344 799914 208632 205741 13228 672411 941921 74327 15141 686735 592377 172295 944379 787361 747524 548440 533596 988768 738475 785609 687534 260546 626858 139752 214392 251360 981284 846570 394817 952251 171662 300687 179639 22413 42563 991605 967533 270841 956337 573630 396571 799170 950081 799908 157957 810147 377272 907892 668435 361226 352733 622006 31663 236317 795034 884352 944901 375920 275946 316578 241433 68428 798802 342188 251061 873302 185533 887736 206343 677226 827509 710945 603755 654594 159650 905597 600799 153761 737210 66489 777589 195111 239907 394355 198986 797533 597881 615620 528020 595886 583932 273078 762291 290834 867196 176253 971109 428661 625604 790422 896903 813355 94080 424215 721841 801106 875704 239377 964447 625496 748911 719728 456212 427190 212491 105761 97594 929323 816391 153477 948504 1770 423087 149219 488256 605062 763574 462029 286792 84767 279461 681037 606001 858973 51630 542236 707940 512699 420914 325035 270031 294815 303499 908570 298371 498307 13198 149070 623576 999433 579156 676914 578032 592562 466776 599810 112690 655367 820935 493407 367663 337803 862785 652727 190485 140597 935067 259644 499860 550580 349197 134597 2794 30666 205681 161386 542760 564576 465969 990452 215528 927942 970623 286000 425862 973821 720052 607619 374359 706946 636357 718213 860811 996310 917305 405032 308022 329518 245400 376037 163797 145158 677193 743380 792475 559914 538255 826172 560047 432596 33795 989026 223752 379475 140930 952951 489595 640827 409068 31341 292081 703454 805964 223113 461360 980523 826301 328716 272712 926033 33753 859701 646666 790017 196078 845710 773654 215749 769882 651713 43975 368868 469442 707609 601836 927346 825698 65929 350605 241334 839037 83731 502672 405963 20167 644274 491229 723529 698610 406413 583403 679796 628273 371369 443917 395895 107746 315043 829262 927984 943262 323103 308843 878610 726022 322915 247781 637558 512822 970219 