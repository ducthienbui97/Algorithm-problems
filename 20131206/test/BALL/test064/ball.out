12111451728155
